From 2d209a4c68627218140f29ca31ee1f01ce476b89 Mon Sep 17 00:00:00 2001
From: Ian Campbell <ijc@hellion.org.uk>
Date: Wed, 26 Nov 2014 15:30:03 +0000
Subject: [PATCH 1/2] arm64: add missing dts entry for X-Gene platform.
Origin: http://lists.infradead.org/pipermail/linux-arm-kernel/2013-December/216347.html

This add missing dts entry for X-Gene reboot driver.

Signed-off-by: Feng Kan <fkan at apm.com>

ijc -- obsoleted by upstream syscon-reboot driver reworking, but that is too
big to backport.

---
 arch/arm64/boot/dts/apm-storm.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

Index: tree-jessie/arch/arm64/boot/dts/apm-storm.dtsi
===================================================================
--- tree-jessie.orig/arch/arm64/boot/dts/apm-storm.dtsi
+++ tree-jessie/arch/arm64/boot/dts/apm-storm.dtsi
@@ -283,6 +283,11 @@
 			interrupts = <0x0 0x4c 0x4>;
 		};
 
+		reboot@17000014 {
+			compatible = "apm,xgene-reboot";
+			reg = <0x0 0x17000014 0x0 0x4>;
+		};
+
 		serial1: serial@1c021000 {
 			status = "disabled";
 			device_type = "serial";
